### ISSUE TITLE
Asymmetric Crosslinking Reagents

### ISSUE THREAD

**Issue by FriedLabJHU** [diogobor/Scout#6]
Related to #1 

We have developed crosslinkers with asymmetric cleavage sites that produce twin doublet peaks which we rely on to isolate CSMs that contain crosslinks. Are there any plans to incorporate these types of crosslinkers in SCOUT? 

![image](https://github.com/diogobor/Scout/assets/86693065/73b19de9-7f58-4d3e-99cc-4fbe1ee7f948)

**Comment by diogobor** [comment]
Yes, we have plans to adapt Scout to perform searches by using this kind of cross-linkers.

As soon as we release a new version with this feature, I'll let you know.
